packages/dcos-integration-test/extra/conftest.py

```diff
@@ -177,6 +177,15 @@ def _wait_for_srouter_slaves_endpoints(self):
             assert "id" in data
             assert data["id"] == slave_id
 
+    @retrying.retry(wait_fixed=2000,
+                    retry_on_result=lambda r: r is False,
+                    retry_on_exception=lambda _: False)
+    def _wait_for_metronome(self):
+        r = self.get('/service/metronome/v1/jobs')
+        if r.status_code == 500 or r.status_code == 504:
```

Comment:
I prefer the being specific like this. I'm just saying a comment / documentation in the code would be nice other than "We seem to be checking two random ports" when someone reads this code.